.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
+  issue, please make sure that this case is not already reported in GitHub as an
+  issue or in StackOverflow as a question.'
+
+---
+
+**Environment**
```

Comment:
We need to add Node version and webpack configuration as well.
It might be a good idea to ask for the full content of the `package.json` because the webpack plugins are usually important for reproducing the problem.

Reply:
Thanks, added those.